src/styles/index.scss

```diff
@@ -1,3 +1,144 @@
+$bodyFont: Roboto, sans-serif;
+$cardColorBorder: #f3f3f3;
+$cardColorProductCode: #616070;
+$cardColorPriceProduct: #060b35;
+$cardColorButtonHover: #00acdc;
+$cardColorButtonBorder: #00acdc;
+$cardColorButtonLink: #fff;
+$cardWidth: 200px;
+$cardHeight: 408px;
+$cardBorderRadius: 5px;
+$buttonHeight: 166px;
+$buttonWidth: 40px;
+$buttonBorderRadius: 5px;
+
 body {
   margin: 0;
+  font-family: $bodyFont;
+}
+
+.card {
+  width: $cardWidth;
+  height: $cardHeight;
+  border: 1px solid $cardColorBorder;
+  border-radius: $cardBorderRadius;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  padding: 0;
+
+  &__top {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    margin: 32px 19px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 16px 4px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $cardColorProductCode;
+    margin: 0 84px 16px 16px;
```

Comment:
instead of setting the horizontal margin to all tags inside the card, you can just add the padding to the card